app/controllers/v1/settings_sets_controller.rb

```diff
@@ -0,0 +1,25 @@
+module V1
+  class SettingsSetsController < ApplicationController
+    before_action :authenticate_user!
+
+    def show
+      render json: current_user.settings_set
```

Comment:
do rozważenia - metoda prywatna zwaracająca current_user.settings_set, bo się to powtarza